vcpkg/users/manifests.md

```diff
@@ -1,140 +0,0 @@
----
-title: Manifest mode reference
-description: Use a manifest with vcpkg to configure libraries on a per project basis.
-ms.date: 01/10/2024
-ms.topic: concept-article
----
-# Manifest mode
-
-vcpkg has two modes for consuming packages - [Classic mode](classic-mode.md) and [Manifest mode](manifests.md). For most users, we recommend Manifest mode.
-
-In Manifest mode, vcpkg creates separate *installed trees* for each project and configuration. This allows separate projects to use different versions of libraries. The [`vcpkg.json` file](../reference/vcpkg-json.md) and optional [`vcpkg-configuration.json` file](../reference/vcpkg-configuration-json.md) form a project's _manifest_. The manifest declares the project's direct dependencies, version constraints, and registries used.
-
-Check out the [manifest CMake example](../examples/manifest-mode-cmake.md) for an example project using CMake and manifest mode.
-
-Check out the [`vcpkg.json` Syntax Reference](../reference/vcpkg-json.md) for a full list of fields and functionality.
-
-## Example
-
-```json
-{
-  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
-  "name": "my-application",
-  "version": "0.15.2",
-  "dependencies": [
-    "boost-system",
-    "libxml2",
-    "yajl"
-  ]
-}
-```
-
-## Features
```

Comment:
@JavierMatosD 
The sections talking about features should be moved into the article you're working on. 
Most of the links in other articles should also be pointed to that future article too.